brax/training/agents/ppo/train.py

```diff
@@ -74,13 +74,46 @@ def f(leaf):
 
 
 def remove_pixels(obs: Union[jnp.ndarray, Mapping]) -> Union[jnp.ndarray, Mapping]:
```

Comment:
last nit: let's keep these functions private since they are in `train.py` after all

Reply:
Moved them :)